docusaurus/docs/reactnative/guides/message_input_customization.mdx

```diff
@@ -229,4 +232,164 @@ const additionalTextInputProps = useMemo(() => {
 >
   ...
 </Channel>
-```
\ No newline at end of file
+```
+
+## Replacing AttachmentPicker With Native Image Picker
+
+Attachment picker opens up as bottom sheet by default.
+It's easy to replace this default attachment picker with a device's native image picker, as shown in screenshots below.
+In this section, we will walk through the integration of native image picker to attach images in message.
```

Comment:
I feel like this explanation its not needed anymore if we go for one of the options I told in the previous comment